_includes/payment-transaction-states.md

```diff
@@ -14,9 +14,9 @@ the course of a purchase. You can find a short description of each state below.
     Examples of possible failures are triggering of anti-fraud protection or if
     the payer reaches the maximum number of attempts for a given payment.
 
-*   `aborted` means that the merchant has aborted the payment before the end user
-    has fulfilled the payment process. This is done by calling the `PATCH` function
-    for abort operations.
+*   `aborted` means that the merchant has aborted the payment before the end
+    user has fulfilled the payment process. This is done by calling the `PATCH`
+    function for abort operations.
```

Comment:
```suggestion
*   `aborted` means that the merchant has aborted the payment before the
    payer has fulfilled the payment process. Aborting a payment is done by
    performing the `update-payment-abort` operation.
```

Reply:
That include wasn't supposed to be a part of this commit, but a great suggestion nonetheless! 👍